images/ose-installer-artifacts.yml

```diff
@@ -34,6 +34,6 @@ owners:
 - aos-install@redhat.com
 konflux:
   vm_override:
-    aarch64: linux-d160/arm64
-    ppc64le: linux-d400-large/ppc64le
+    x86_64: linux-d160-m2xlarge/amd64
+    aarch64: linux-d160-m2xlarge/arm64
```

Comment:
ppc64le arch override was overritten with this change, would need that
`ppc64le: linux-d400-large/ppc64le`